docs/shone/examples/opacities.rst

```diff
@@ -0,0 +1,267 @@
+
+    download_molecule(
+        molecule_name='H2O',
+        temperature_range=[2500, 2500],
+        pressure_range=[-6, -6]
+    )
+
+or for a specific isotopologue:
+
+.. code-block:: python
+
+    from shone.opacity.dace import download_molecule
+
+    download_molecule(
+        isotopologue='1H2-16O',
+        temperature_range=[2500, 2500],
+        pressure_range=[-6, -6]
+    )
+
+These methods download the opacity grids from DACE, and store them in a netCDF file
+in your home directory with the name ``.shone/``.
+
+Example opacity file
+--------------------
+
+It is often useful to have a small, synthetic opacity file for simple demos and testing.
+You can generate one with `~shone.opacity.generate_synthetic_opacity()`:
+
+.. code-block:: python
+
+    from shone.opacity import generate_synthetic_opacity
+
+    opacity = generate_synthetic_opacity()
+
+You can also lookup and load opacities within the cache like so:
+
+.. code-block:: python
+
+    from shone.opacity import Opacity
+
+    # load the one opacity file:
+    opacity = Opacity.load_species_from_name('synthetic')
+
+Opening opacities
+-----------------
+
+Opacities are loaded and interpolated with the `~shone.opacity.Opacity` class.
+
+.. code-block:: python
+
+    from shone.opacity import Opacity
+
+We can check which species are already chached and available on your
+machine using `~shone.opacity.Opacity.get_available_species()`:
+
+.. code-block:: python
+
+    Opacity.get_available_species()
+
+This will return a table of available opacity grids on disk.
+
+Let's load the opacity created by `~shone.opacity.generate_synthetic_opacity()`
+(see the step above):
+
+.. code-block:: python
+
+    opacity = Opacity.load_species_from_name('synthetic')
+
+The `~shone.opacity.Opacity` object contains the opacity grid as a `~xarray.DataArray`
+in its `grid` attribute. You can see the dimensions of the grid with:
+
+.. code-block:: python
+
+    >>> print(opacity.grid.coords)
+    Coordinates:
+      * wavelength   (wavelength) float64 0.5 0.5012 0.5023 ... 4.977 4.988 5.0
+      * temperature  (temperature) int32 200 400 600 800 1000
+      * pressure     (pressure) float64 1e-06 10.0
+
+The coordinates in the `~xarray.DataArray` are wavelength in microns,
+temperature in K, and pressure in bar. To learn to use the xarray API
+directly on the grid attribute, refer to the xarray docs on `indexing
+and selecting data <https://docs.xarray.dev/en/stable/user-guide/indexing.html>`_
+and `interpolating
+<https://docs.xarray.dev/en/stable/user-guide/interpolation.html>`_.
+
+You can inspect the opacities from one temperature and pressure slice like so:
+
+.. code-block:: python
+
+    import matplotlib.pyplot as plt
+
+    opacity_sample = opacity.grid.sel(
+        dict(
+            pressure=10,  # [bar]
+            temperature=200  # [K]
+        )
+    )
+
+    plt.semilogy(
+        opacity_sample.wavelength, opacity_sample.opacity,
```

Comment:
`opacity_sample.opacity` is wrong. It should be `opacity_sample`, equal to what is the source code of the plot that comes after this cell.